### ISSUE TITLE
Support jigsaw module

### ISSUE THREAD

**Issue by SchweinchenFuntik** [JetBrains/Exposed#1088]
when adding the exposed dependency, when the application starts, it starts giving an error (see below), trying to exclude nothing helps
```
Error occurred during initialization of boot layer
java.lang.LayerInstantiationException: Package kotlin in both module mymodule.merged.module and module kotlin.stdlib
```

**Comment by SchweinchenFuntik** [comment]
it was my mistake, I did not exclude correctly: I did `exclude(module = "kotlin.stdlib")` and `exclude(group = "org.jetbrains.kotlin")`. Only the option with the module was needed.

But the question remains: Will exposed be split into modules?

**Comment by Tapac** [comment]
@SchweinchenFuntik , it's already split into [modules](https://github.com/JetBrains/Exposed/wiki/LibDocumentation#exposed-0181-and-higher). 
Do you have any problems with the current modules?

**Comment by SchweinchenFuntik** [comment]
no, I managed to start in compatibility mode, but will there be support for them?

**Comment by SchweinchenFuntik** [comment]
@Tapac I'm not talking about these [modules](https://github.com/JetBrains/Exposed/wiki/LibDocumentation#exposed-0181-and-higher), but about the jigsaw project

**Comment by HollisMeynell** [comment]
> @SchweinchenFuntik , it's already split into [modules](https://github.com/JetBrains/Exposed/wiki/LibDocumentation#exposed-0181-and-higher). Do you have any problems with the current modules?

Java 9 Platform Module System, using JPMS allows specifying provided services when developing dependency libraries, which can reduce dependency conflicts between projects. Additionally, it enables jlink to work, allowing for a smaller packaged product by customizing the JREwill there be support for them?

**Comment by HollisMeynell** [comment]
Any progress?
